index.html

```diff
@@ -3413,37 +3413,37 @@ <h3>Syntactic Sugar</h3>
         <h3>Media Types</h3>
 
         <p>
-Media types as defined in [[RFC6838]] serve a useful purpose with verifiable
-credentials, especially regarding identification of a particular syntax that is
-in use with a verifiable credential.
+Media types, as defined in [[RFC6838]], identify the syntax used to express a
+<a>verifiable credential</a> as well as other useful processing guidelines.
         </p>
         <p>
-Syntaxes SHOULD be identified by a media type, and certain conventions as
-outlined in this section SHOULD be followed when defining or using media types
-with verifiable credentials.
+Syntaxes used to express the data model in this specification SHOULD be
+identified by a media type, and conventions outlined in this section SHOULD be
+followed when defining or using media types with <a>verifiable credentials</a>.
         </p>
         <p>
-Any media types associated with the core data model are listed in the section on
-<a href="#iana-considerations">IANA Considerations</a> for registration with
-IANA.
+Any media types associated with the core data model are listed in the Section
+<a href="#iana-considerations"></a>.
         </p>
         <p class="issue" data-number="1060">
-Presence of the term "verifiable" (or a shortened version) in the media type may
-or may not imply a proof.  Original proposed language is tracked in the issue
-corresponding to this note.
+The VCWG is currently discussing whether the presence of the term "verifiable"
+(or a shortened version) in the media type may or may not imply a proof. The
+Working Group is seeking implementation experience related to the use of media
+types and the expectation related to whether or not a particular media type
+is expected to be secure with a particular proof.
         </p>
         <p>
-Use of the term `credential` in a media type related to a syntax of verifiable
-credentials <i>without</i> the corresponding use of `verifiable` as described
-above <i>does not</i> imply the presence of a proof with the
-<a href="#dfn-credential">credential</a>.
+Use of the term "credential" in a media type related to a syntax of
+<a>verifiable credentials</a> <em>without</em> the corresponding use of
+"verifiable" as described above <em>does not</em> imply the presence of a
+proof with the <a>credential</a>.
         </p>
         <p>
-At the time of this writing, there is one media type associated with the core
-data model: `application/credential+ld+json`. Other specifications such as
-[[VC-JWT]] define additional media types as may be required to allow for proper
-interpretation of the syntax of verifiable credentials encoded in the syntax
-described by the respective specification.
+There is one media type associated with the core data model:
+`application/credential+ld+json`. Other specifications, such as [[?VC-JWT]],
```

Comment:
I believe that since the plan is for VC-JWT to follow roughly the same recommendation-track timeframe as VC Data Model v2, we can probably keep the references to it as normative.

```suggestion
`application/credential+ld+json`. Other specifications, such as [[VC-JWT]],
```

Reply:
I guess this is a question of Editorial style. What I prefer to do is make *non-normative references* to external specifications non-normative. That is, if you're just mentioning another specification in a non-normative way, the reference should be non-normative as well. I try to do this because as text is shuffled to-and-fro within the specification, you want the normative statements to be the ones that make a specification reference normative.

I expect that this is not done consistently in the vc-data-model spec at present (and I was going to do a pass right before CR to clean this up).  So, I'd prefer to keep this reference non-normative as we have other references in the spec today, to VC-JWT, that are normative.

All that said, I'm wondering if we want any normative references to VC-JWT or VC-DATA-INTEGRITY. I would expect the WG to say "yes, we do"... but I wonder if that's what is best for the specification. The data model can technically, and debatably, stand on its own without dependence on any "securing" specification. The only place where this might not be true is the [Proof Formats](https://w3c.github.io/vc-data-model/#proof-formats) section, which some are arguing to remove from the specification.

In any case, just raising that as a consideration. I would prefer to not take this change, but wouldn't be averse if others felt like the reference should be normative (at which point, the editorial rule that we use wrt. normative vs. non-normative references would need to be consistently applied throughout the specification).

Reply:
I don't think you need to understand `vc-jwt` to understand `credential+ld+json` or any `*+ld+json`.

I believe the reference should remain informative.

Reply:
makes sense to me.